Controllers/CartController.cs

```diff
@@ -177,14 +177,59 @@ public CartController(BangazonContext ctx)
                 openOrder.DateCompleted = DateTime.Now;
```

Comment:
You can remove the `[fromRoute]int id` in the `CompleteOrder` method because it is not used.